src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -180,14 +180,17 @@ def process_list(
             echo.echo_report(f'Using {percent_load * 100:.0f}% of the available daemon worker slots.')
 
 
-@verdi_process.command('show')
+@verdi_process.command('show', short_help='Show details for the given process(es).')
 @arguments.PROCESSES()
 @options.MOST_RECENT_NODE()
 @decorators.with_dbenv()
 def process_show(processes, most_recent_node):
-    """Show details for one or multiple processes."""
+    """Show details for one or multiple processes given the primary key(s) (PK)."""
```

Comment:
I wouldn't add primary key(s) (PK). The pk is just one way of identifying a node. All options/arguments that allow identifying an ORM entity (`Computer`, `Node`, `User`) they can all be identified by their PK, UUID or label. The pk might be the most common one, but I don't think we should write it like this as it may falsely suggest that it is the only possible identifier

Reply:
Is it okay if we change it like in `kill` ?
```
def kill_processes(processes):
    """Show details for one or multiple processes.

    Show the details of the process(es) given by PROCESSES. 
    """
```
(removing short help)

And secondly, I am not sure what valid identifiers are from the verdi command line. Can we maybe add a link to the doc?
```
def kill_processes(processes):
    """Show details for one or multiple processes.

    Show the details of the process(es) given by the identifier PROCESSES. 
    See https://aiida.readthedocs.io/projects/aiida-core/en/stable/topics/cli.html#topics-cli-identifiers for valid identifiers.
    """
```
Or explicitly write them out
```
def kill_processes(processes):
    """Show details for one or multiple processes.

    Show the details of the process(es) given by the identifier PROCESSES. Valid identifiers are: PK, UUID or label.
    """
```

Reply:
> Can we maybe add a link to the doc?

Here you go: https://aiida.readthedocs.io/projects/aiida-core/en/stable/topics/cli.html#topics-cli-identifiers

But honestly, I don't want to have to start adding this to all the commands. This is a common feature that applies to all commands where ORM entities can be referenced. It would be way too repetitive to add this everywhere.

Reply:
I see your point adding this information is very redundant. What do you think about changing the PROCESSES `click.Parameter` ? 
```python
PROCESSES = OverridableArgument('processes', nargs=-1, type=types.ProcessParamType(),
                                metavar="[PK|UUID|Label]...")
```
resulting help page:
```
Usage: verdi process show [OPTIONS] [PK|UUID|Label]...

  Show details of processes.

  Show the details of the process(es) given by the identifier(s).

Options:
  -M, --most-recent-node          Select the most recently created node.
  -v, --verbosity [notset|debug|info|report|warning|error|critical]
                                  Set the verbosity of the output.
  -h, --help                      Show this message and exit.
```
I added `...` to specify that a list of arguments can be given but that might not be clear.

Reply:
Ah wow the `...` is even specified in the doc. I see that this goes a bit beyond the PR. I would make maybe a separate discussion in an issue after looking at this in more depth

Reply:
But I think `[PROCESSEs]...` would be actually the correct way according to the documentation. Should I change it to this for now?

Reply:
> Ah wow the ... is even specified in the doc. I see that this goes a bit beyond the PR. I would make maybe a separate discussion in an issue after looking at this in more depth

I agree, this is getting way out of scope. The important part in this PR is to make sure all process commands properly error if no process is specified. I am fine with adding the missing `most-recent-node` option for the one command. The discussion on short help messages is already getting out of scope as well, but we could include it if we can come to an agreement, otherwise we should maybe keep it simple and leave that out.

> But I think [PROCESSES]... would be actually the correct way according to the documentation. Should I change it to this for now?

Yes, this is the correct notation and it is automatically generated by click. I prefer not to change it manually unless we really have to. Having these things generated automatically guarantees a large degree of consistency across all commands

Reply:
I see now it is working again. I probably disabled somehow unintentionally or it is getting too late. Removed any manual changes.